packages/@aws-cdk/aws-lambda/README.md

```diff
@@ -277,6 +277,58 @@ correct log retention period (never expire, by default).
 *Further note* that, if the log group already exists and the `logRetention` is not set, the custom resource will reset
 the log retention to never expire even if it was configured with a different value.
 
+### FileSystem Access
+
+You can configure a function to mount an Amazon Elastic File System (Amazon EFS) to a
+directory in your runtime environment with the `filesystems` property. To access Amaozn EFS
+from lambda function, the Amazon EFS access point will be required. 
+
+Amazon EFS _access points_ are application-specific entry points into an EFS file system that
+make it easier to manage application access to shared datasets. Access points can enforce a user
+identity, including the user's POSIX groups, for all file system requests that are made through
+the access point. Access points can also enforce a different root directory for the file system
+so that clients can only access data in the specified directory or its subdirectories. See
+[Working with Amazon EFS Access Points](https://docs.aws.amazon.com/efs/latest/ug/efs-access-points.html) for more details.
+
+The following sample enables the lambda function to mount the Amazon EFS filesystem via the `accessPoint` to `/mnt/msg` in the runtime environment. We have to create an access point in Amaozn EFS and make sure Lambda function has
+read write access to it.
+
+```ts
+// create a new Amaozn EFS filesystem
+const fileSystem = new efs.FileSystem(stack, 'Efs', { vpc });
+
+// create a new access point from the filesystem
+const accessPoint = fileSystem.addAccessPoint('AccessPoint', {
+  // set /mnt/msg as the root of the access point
+  path: '/mnt/msg',
+  // as /mnt/msg does not exist, the efs will create the directory with the following createAcl
+  createAcl: {
+    ownerUid: '1001',
+    ownerGid: '1001',
+    permissions: '755',
+  },
+  // enforce the POSIX identity so lambda function will access with this identity
+  posixUser: {
+    uid: '1001',
+    gid: '1001',
+  },
+});
+
+const fn = new lambda.Function(stack, 'MyLambda', {
+  code,
+  handler,
+  runtime,
+  vpc,
+  securityGroups: fileSystem.connections.securityGroups,
```

Comment:
Now that the `filesystem` is exposed on the `accessPoint` you should be able to create the security group and "connect it" to the filesystem.

Reply:
@jogold  thanks for the chime in. I found `AccessPoint` can only expose the `IFileSystem` in this case. Any suggestions?

Reply:
`IFileSystem` extends `ec2.IConnectable` so you have access to the `connections` object

Reply:
Do you mean something like this?

```
accessPoint.filesystem.connections.AllowToDefaultPort(function)
```
However, if  we `fn.node.addDependency(accessPoint.filesystem)` , this will create circular dependency.

suggestions?

Reply:
more something like `accessPoint.filesystem.connections.allowDefaultPortFrom(function)`. About the circular dependency, did you test this? What if you revert the relation `function.connections...`? If there's effectively a circular dependency then it would also be present outside of this construct, when a user connects it...